### ISSUE TITLE
albumentation 

### ISSUE THREAD

**Issue by ruchika61** [qubvel/segmentation_models#411]
Hi 
i am getting this error:

  File "C:\Users\dell\Anaconda3\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 704, in runfile
    execfile(filename, namespace)

  File "C:\Users\dell\Anaconda3\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 108, in execfile
    exec(compile(f.read(), filename, 'exec'), namespace)

  File "E:/MCSS/segmentation_models-master/load_dataset.py", line 292, in <module>
    preprocessing=get_preprocessing(preprocess_input),

  File "E:/MCSS/segmentation_models-master/load_dataset.py", line 240, in get_preprocessing
    A.Lambda(image=preprocessing_fn),

AttributeError: module 'albumentations' has no attribute 'Lambda'
please suggest some solution for this error

_Originally posted by @ruchika61 in https://github.com/qubvel/segmentation_models/issues/408#issuecomment-713126589_

**Comment by xiankgx** [comment]
I don't think this should be here. But try upgrading/downgrading albumentations.

**Comment by ruchika61** [comment]
Hi
 Solved this error after upgrading
Thanks
